frameworks/PHP/php/php-caddy.dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:22.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -yqq && apt-get install -yqq software-properties-common > /dev/null
+RUN LC_ALL=C.UTF-8 add-apt-repository ppa:ondrej/php > /dev/null && \
+    apt-get update -yqq > /dev/null && apt-get upgrade -yqq 
+
+RUN apt-get install -yqq nginx git unzip curl \
```

Comment:
Probably don't need/want `nginx` installed.

Reply:
jeje, 
yes of course